deploy/charts/trust-manager/values.yaml

```
@@ -101,6 +101,18 @@ app:
     # -- If false, disables the default seccomp profile, which might be required to run on certain platforms
     seccompProfileEnabled: true
 
+secretTargets:
+  # -- If set to true, enable writing trust bundles to Kubernetes Secrets as a target
+  enabled: false
```

suggestion: I think it'd help to add a comment here indicating that users need to do something else in addition to setting this to true.

```suggestion
  # -- If set to true, enable writing trust bundles to Kubernetes Secrets as a target. trust-manager can only write to secrets which are explicitly allowed - see either authorizedSecrets or authorizedSecretsAll.
  enabled: false
```